ts/protocol/otf_message_handler.py

```diff
@@ -64,6 +64,9 @@ def create_predict_response(
     :param code:
     :return:
     """
+    if str(os.getenv("LOCAL_RANK", 0)) != "0":
```

Comment:
This low level method should not be concerned about checking the LOCAL_RANK env environment. We should just not call it when its not "0".

Reply:
I see the intention here. I still think this is not the preferred place to put this check. The function creating a response from some arguments should not need to know about the concept of ranks. In fact we already have a central point to check for the rank in this PR which is [here](https://github.com/pytorch/serve/blob/e7dbf4562f9aac8d6cc07aa6d36e4c017e8c919c/ts/model_service_worker.py#L183).
The only point where this is missing is [here](https://github.com/pytorch/serve/blob/e7dbf4562f9aac8d6cc07aa6d36e4c017e8c919c/ts/protocol/otf_message_handler.py#L362). Looking at this I think we should move send_intermediate_predict_response out of ts.protocol.otf_message_handler (Users should not need deal with the modules containing our comms between fe/be. What if we drop otf protocol?). My first instinct is to move it under ts.handler_utils. What do you think?

Reply:
AFAIK we do not guarantee bc as of now. Its probably time to plan out our bc strategy for future release as we're running out or 0.X version numbers. Anyways, if your concern is very high you can leave a wrapper in otf_message_handler that fires a deprecation warning and then calls the one in handler_utils until next release. cx will only get worse if we let the code rot.